src/renderer/renderer.es6

```diff
@@ -201,7 +201,7 @@ export default function Renderer(phonemes, pitch, mouth, throat, speed, singmode
       }
       // unvoiced
       off = pitch ^ 255 & 0xFF; // unsigned char
-      let mem53 = tab48426[hibyte] & 0xFF; // unsigned char
+      let value0 = sampledConsonantValues0[kind] & 0xFF; // unsigned char
```

Comment:
When renaming `mem53` to `value0` and `hibyte` to `kind`, please pay attention to do this on all occasions.

I have amended this PR on my side now in c56978571af76a22210926e202d74f17b17518c5.

Reply:
Sorry, those names remained from a bigger patch I had locally -- now submitted as https://github.com/discordier/sam/pull/14